core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -1177,16 +1169,20 @@ private[spark] class AppStatusListener(
       }
 
       cleanupCachedQuantiles(key)
+      key
     }
 
+    // Delete summaries in one pass, as deleting them for each stage is slow
+    val totalSummariesDeleted = kvstore.removeAllByKeys(
```

Comment:
Unused variable.

Reply:
excellent - yes, this used to be where I logged the count, thanks for catching!